datahub/metadata/migrations/0063_update_trade_agreements.py

```diff
@@ -0,0 +1,23 @@
+from pathlib import PurePath
+
+from django.db import migrations
+
+from datahub.core.migration_utils import load_yaml_data_in_migration
+
+
+def load_trade_agreements(apps, _):
+    load_yaml_data_in_migration(
+        apps,
+        PurePath(__file__).parent / '0063_update_trade_agreements.yaml'
+    )
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('metadata', '0062_update_services'),
```

Comment:
```suggestion
        ('metadata', '0062_update_trade_agreements'),
```

This should fix your tests